.github/workflows/release-test.yml

```diff
@@ -116,6 +116,11 @@ jobs:
       if: ${{ matrix.pytest_mark == 'not iotlab_creds' }}
       run: |
         sudo RIOT/dist/tools/tapsetup/tapsetup -c 11
+    - name: Install native dependencies
+      if: ${{ matrix.pytest_mark == 'not iotlab_creds' }}
+      run: |
+        sudo apt-get update
+        sudo apt-get install lib32asan5
```

Comment:
waitaminute, does this not need a "--yes" or similar?

Reply:
```suggestion
        sudo apt-get -y install lib32asan5
```

Reply:
Not for Github Actions afaik. Never used it at least and it is not listed [in the doc](https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners). Also, as you can see in https://github.com/miri64/RIOT/runs/6552855739?check_suite_focus=true, it runs just fine without it ;-).

Reply:
ok :)

Reply:
I guess someone at Github anticipated this common tripwire ;-).